### DISCUSSION TITLE
Is a Quick Viewer available to check out each search result?

### DISCUSSION THREAD

**Discussion by raindropsfromsky** [Flow-Launcher/Flow.Launcher#1564] [Q&A] [answered]
Many popular file managers have a QuickViewer panel, which shows the preview of the file under focus.
Is this facility available in Flow Launcher?

**Note:** 
I was not sure if this should be posted under the "plugin request" section.
The Q&A seems to be a "catch all" section. So posting here!

**Comment by VictoriousRaptor** [comment]
It's working in progress.

**Comment by TheBestPessimist** [comment, marked as answer]
I believe this was released in V1.10: https://github.com/Flow-Launcher/Flow.Launcher/releases/tag/v1.10.0
